scripts/prepare_lambda_deployment.py

```diff
@@ -0,0 +1,73 @@
+from argparse import ArgumentParser
+import json
+import logging
+from pathlib import Path
+import shutil
+import sys
+
+from azul import config
+from azul.files import write_file_atomically
+from azul.logging import configure_script_logging
+
+log = logging.getLogger(__name__)
+
+
+def transform_tf(input_json):
+    # Using the default provider makes switching deployments easier
+    del input_json['provider']
+
+    assert 'variable' not in input_json
+    input_json['variable'] = {'role_arn': {}}
+
+    input_json['output']['rest_api_id'] = {
+        'value': '${aws_api_gateway_rest_api.rest_api.id}'
+    }
+
+    for func in input_json['resource']['aws_lambda_function'].values():
+        assert func['source_code_hash'] == '${filebase64sha256("./deployment.zip")}'
+        func['source_code_hash'] = '${filebase64sha256("${path.module}/deployment.zip")}'
+        assert func['filename'] == "./deployment.zip"
+        func['filename'] = "${path.module}/deployment.zip"
+
+    # TODO: Remove when https://github.com/aws/chalice/issues/1237 is merged
```

Comment:
Could you create an Azul ticket (to periodically check on the Chalice issue) and add a link here, in addition to the Chalice ticket link?

Reply:
I moved the description out of the code and into the ticket as well.